docs/discriminators.md

```diff
@@ -31,12 +31,25 @@ instances.
 
 ### Discriminator keys
 
-The way mongoose tells the difference between the different
-discriminator models is by the 'discriminator key', which is
-`__t` by default. Mongoose adds a String path called `__t`
-to your schemas that it uses to track which discriminator
-this document is an instance of.
+The way Mongoose tells the difference between the different discriminator models is by the 'discriminator key', which is `__t` by default.
+Mongoose adds a String path called `__t` to your schemas that it uses to track which discriminator this document is an instance of.
```

Comment:
is this meant to be on the same line? (previous line is missing double spaces to make it a new line)